### ISSUE TITLE
Cortana flyout not flush with top of screen when Start Menu is full screen

### ISSUE THREAD

**Issue by asmitde** [TheJoeFin/Windows10-Community#6]
1. Set Start Menu to be full screen from Settings.
2. Click on Cortana to open flyout.

The top of the flyout leaves a little room from the top of the screen. It should be at full screen height.

FBH: https://aka.ms/AA1t830

<img width="677" alt="cortana issue" src="https://user-images.githubusercontent.com/23269916/42598775-2e3d77c8-852b-11e8-8eed-af38db0b0cef.png">

**Comment by TheJoeFin** [comment]
This is on the insider build right?

**Comment by asmitde** [comment]
Yes, sorry, forgot to mention. Build 17711. Issue started when they introduced the new wide UI for Cortana.
